### DISCUSSION TITLE
remix docs scrolls to top with theme change

### DISCUSSION THREAD

**Discussion by lkrbalci** [remix-run/remix#6291] [General]
toggling between dark and light color themes in the website's docs section causes the page to scroll to the top. That is not a good UX I think, as it makes the user search where he or she was all over the page.

**Comment by machour** [comment]
Thank you for reporting this, I've filled an issue in the (private) website repository.

**Reply by machour** [reply]
@lkrbalci this is now fixed. We added `preventScrollReset` on the `Form` 👌🏼

**Reply by lkrbalci** [reply]
I can confirm it works 👍 thanks :)
